src/main/resources/io/viash/platforms/nextflow/channel/_parseParamList.nf

```diff
@@ -81,20 +83,28 @@ def _parseParamList(param_list, Map config) {
     data = _splitParams(data, config)
     [id, data]
   })
-  
+    
   // The paths of input files inside a param_list file may have been specified relatively to the
   // location of the param_list file. These paths must be made absolute.
   if (paramListPath) {
+
+    // functionality to resolve a single path (as String) relative to the param_list file
+    // but only if the file does not specify a remote path 
+    def resolveIfNotRemote = { path, parentPath -> 
+      if (path instanceof String && FileHelper.getUrlProtocol(path) == null) {
+        return parentPath.resolveSibling(path) 
+      }
+      return path
+    }
```

Comment:
Can I propose defining this function outside of the `_parseParamList()` function?